app/scripts/controllers/savings/ViewSavingDetailsController.js

```diff
@@ -108,6 +108,10 @@
 
             resourceFactory.savingsResource.get({accountId: routeParams.id, associations: 'all'}, function (data) {
                 scope.savingaccountdetails = data;
+                resourceFactory.groupResource.get({groupId: scope.savingaccountdetails.groupId, associations: 'all'}, function (data) {
```

Comment:
why do we need to get the group with "all" associations ? You only need the group name